package_manifest.yaml

```diff
@@ -500,14 +513,6 @@ foreman_nonscl_packages:
         - el8-puppet-6
   hosts:
     foreman-bootloaders-redhat: {}
-    foreman-installer:
-      releasers:
-        - "{{ nightly_releaser }}"
-      build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
-      nightly_package_tito_releaser_args:
-        - "jenkins_job=foreman-installer-develop-source-release"
-        - "jenkins_url=https://ci.theforeman.org"
-      build_package_tito_builder: "tito.builder.FetchBuilder"
     foreman-release-scl: {}
     foreman-selinux:
```

Comment:
Why isn't this part of it? The installer depends on it and AFAIK is the only package that depends on foreman-selinux (besides katello-selinux).

Reply:
Will do